commands/device/stop-application.ts

```diff
@@ -0,0 +1,36 @@
+
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import helpers = require("./../../helpers");
+
+export class StopApplicationOnDeviceCommand implements ICommand {
+
+	constructor(private $devicesServices: Mobile.IDevicesServices,
+		private $errors: IErrors,
+		private $stringParameter: ICommandParameter,
+		private $staticConfig: Config.IStaticConfig,
+		private $options: IOptions) { }
+
+	allowedParameters: ICommandParameter[] = [this.$stringParameter];
```

Comment:
stringParameter will be disregarded, as this command implements `canExecute` method. Are we sure we want this?